flipperdevices/flipperzero-firmware
# Add Fleet workspace configuration files

This adds some basic configuration for the [JetBrains Fleet IDE](https://www.jetbrains.com/fleet/). It makes things a little nicer for any developers working in Fleet (like myself), without changing anything for anyone else (other than I suppose making the project slightly larger).

# What's new

- `.fleet` directory, allows for workspace configuration within Fleet
    - `run.json` file, ported everything from `.vscode/tasks.json` over
    - ~~`settings.json` file, basic Fleet toolchain configuration~~ reverted as of 5628a07

# Verification 

- Open the project in Fleet and try the run commands
- Open the project in Visual Studio Code or another IDE to confirm that nothing has changed

# Checklist (For Reviewer)
